contrib/docker/README.md

```diff
@@ -0,0 +1,86 @@
+## Docker
+
+Docker can be used to run a node and enable containerization abilities.
+The following propose a base image to deploy a Dogecoin Core node.
+
+> **Warning:** Do not use for production.
+
+### Table of content
+
+1. [Install Docker](#install-docker)
+2. [How to use this image](#how-to-use-this-image)
+3. [Docker run syntax](#docker-run-syntax)
+4. [Docker-compose example](#docker-compose-example)
+
+### Install Docker
+
+Follow official [Get Docker](https://docs.docker.com/get-docker/) instructions to install Docker for your own system.
+
+For security reasons, to avoid running `docker` command with `sudo`, add your user in the `docker` group. See [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/).
+
+### How to use this image
+
+> **Warning:** Even if it's optional, mapping of the data directory with a volume is highly recommended, to avoid redownloading the blockchain at each container creation.
+
+Example to launch a node and start using `dogecoin-cli` :
+```bash
+# With the repository, go to Dockerfile & docker-entrypoint.py location
+$ cd contrib/docker
+
+# Alternatively, download Dockerfile & entrypoint script
+$ wget https://raw.githubusercontent.com/dogecoin/dogecoin/master/contrib/docker/docker-entrypoint.py
+$ wget https://raw.githubusercontent.com/dogecoin/dogecoin/master/contrib/docker/Dockerfile
```

Comment:
The benefit is to be able to run your daemon by downloading only 2 files. 

Wouldn't be equal to `latest` release ? `1.14.4-maint` & `master` are at the same commit

Reply:
I'm not able to understand the implication it creates to add this signature file & how a central authority is needed, because at least I do not understand the flow used to check hashes in the https://github.com/docker-library/official-images#security examples.

Gitian is the decentralized way to check hashes, you can check [gitian.sigs](https://github.com/dogecoin/gitian.sigs), but are people doing it & wouldn't it be a way to have more people checking hashes to add the list to releases ? It should give more incentive.